src/components/AppVersions/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from "@saleor/macaw-ui";
+
+const useStyles = makeStyles(
+  theme => ({
+    container: {
+      display: "flex",
+      justifyContent: "flex-end"
+    },
+    versionItem: {
+      [theme.breakpoints.down("md")]: {
+        fontSize: theme.spacing(1.75)
+      },
+      [theme.breakpoints.up("md")]: {
+        fontSize: theme.spacing(2)
+      },
+      color: "rgba(40, 35, 74, 0.6)",
```

Comment:
Can't we use color from theme? I guess it should be defined there.

Reply:
I didn’t find it in the theme. I checked as RGBA and Hex.